### PR TITLE
Add description column to group table

### LINKED ISSUE #2529: Group description
It would be nice if the Group table would allow for some kind of longer description (for example including a reference or url or whatever) in a dedicated column.

### PR DESCRIPTION
**Pull request by jadalilleboe**
Closes #2529 

Adds a column to the group table for a longer description of the group.

### CODE REVIEW

File: skyportal/tests/api/test_groups.py [hunk 1]
```diff
@@ -11,7 +11,11 @@ def test_token_user_create_new_group(super_admin_token, super_admin_user):
...
         "groups",
-        data={"name": group_name, "group_admins": [super_admin_user.id]},
+        data={
+            "name": group_name,
+            "description": "Group description",
+            "group_admins": [super_admin_user.id],
+        },
         token=super_admin_token,
     )
     assert status == 200
```
Comment (mcoughlin) [inline skyportal/tests/api/test_groups.py:21]: I would add to this test you get your correct description back

**Review by mcoughlin — APPROVED** [review]
@jadalilleboe just one minor comment, otherwise fine for me. Also, will need to add this to the frontend group creation, but that can be in the next PR.

File: skyportal/tests/api/test_groups.py (hunk 1 above)
Comment (jadalilleboe) [reply skyportal/tests/api/test_groups.py:22]: added

Comment (mcoughlin) [reply skyportal/tests/api/test_groups.py:22]: ok good for me once the tests are green.

**Comment by jadalilleboe** [comment]
I think I will move this to #2678, it seems small enough to be in 1 PR and the tests are passing there.

**Pull request closed without merging**